kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -321,7 +321,12 @@ def empty_like(x, shape):
     # something more standard.
     return x.new(*shape)
 
-backend = namedtuple('backend', ['name', 'cdgmm', 'modulus', 'subsample_fourier', 'fft', 'Pad', 'unpad', 'empty_like'])
+
+def concatenate(arrays):
+    return torch.stack(arrays, axis=-3)
```

Comment:
This seems so much simpler than what we were doing beforehand!

Reply:
Indeed it is.